FrontEnd/src/BookingPage/BookingPage.jsx

```diff
@@ -113,12 +128,12 @@ function BookingForm({ userData }) {
         />
         <button type="submit">Send Request</button>
       </form>
-      <Modal show={isModalShown} onClose={toggleModal}>
+      <Modal isShown={isBookingFormModalShown} onClose={toggleModal}>
         <p className="thankYou">
           Thank you for requesting an appointment with {post.vendorUsername}
         </p>
       </Modal>
-      {invisibleComponent && <NotificationsPage vendorId={post.userId} />}
+      {invisibleComponent && <NotificationsPage vendorID={post.userId} />}
```

Comment:
When will the `invisibleComponent` be true?

Reply:
It will never be true since it's not supposed to show. It's only drawing data from booking form as prop